File: Cs/Solution/CanAliceWin.cs
namespace Solution
{
	public partial class Solution
	{
		public bool CanAliceWin(int[] nums)
		{
			int digit_1_sum = 0;
			int sum = 0;
			foreach (int n in nums)
			{
				if (n < 10)
				{
					digit_1_sum += n;
				}
				sum += n;
			}
			return sum != (digit_1_sum + digit_1_sum);
		}
	}
}

File: Rust/src/solution/can_alice_win.rs
use super::base::Solution;

impl Solution {
    pub fn can_alice_win(nums: Vec<i32>) -> bool {
        let sum: i32 = nums.iter().sum();
        let sum_single_digits: i32 = nums.iter().filter(|&&n| n < 10).sum();
        sum != sum_single_digits * 2
    }
}
